saleor/graphql/schema.graphql

```diff
@@ -102,7 +102,9 @@ type Attribute implements Node {
   name: String
   productType: ProductType
   productVariantType: ProductType
+  translations(before: String, after: String, first: Int, last: Int): AttributeTranslationCountableConnection
```

Comment:
Do we want to expose `translations` field as well for each model? Having only `translation(languageCode)` seems fine.

Reply:
Now that's an interesting one, I didn't plan on adding this.